src/features/ToDos/ToDoItem/ToDoItem.module.css

```diff
@@ -0,0 +1,37 @@
+.toDoText {
+  display: inline-block;
+  margin-left: 1rem;
+  color: #a3a5b2;
+}
+
+.crossed {
+  text-decoration: line-through;
+  color: #36d7b7;
+}
+
+.isCompletedCheckBox {
```

Comment:
A very confusing classname. Seems like it should be applied only on checked checkbox